### PR TITLE
v1.0.3 updating for service coverage

### PR DESCRIPTION
**Pull request by ErichDylus**
Updating insurance references to service coverage and related edits, resolving Issue 10 (https://github.com/api3dao/api3-whitepaper/issues/10) with ChainAPI link, other minor edits.

### CODE REVIEW

**Comment by bbenligiray** [comment]
Thanks, will check it out over the weekend

File: api3-whitepaper.tex [hunk 1]
```diff
@@ -106,20 +106,20 @@ \section{Introduction}
...
 
 In the case of a malfunction, the dAPI user will be able to claim compensation up to a pre-negotiated amount from the staking pool.
-Kleros~\cite{kleros:2019}, an on-chain dispute resolution protocol, will be used to decide if the claim is to be paid out based on the presented evidence.
+Kleros~\cite{kleros:2019}, an on-chain dispute resolution protocol, will be used to decide if the claim is to be paid out pursuant to the service coverage terms and conditions based on the presented evidence.
 This will incentivize stakers to actively participate in governance to ensure that dAPIs are being managed transparently and in a way that minimizes security risks.
-Successful governance---successfully monetizing dAPIs while avoiding mistakes that will result in paying out insurance claims---will be rewarded in API3 tokens, which will create a positive feedback loop that will continually improve governance.
+Successful governance---successfully monetizing dAPIs while avoiding mistakes that may result in paying out service coverage claims---will be rewarded in API3 tokens, which will create a positive feedback loop that will continually improve governance.
```
Comment (bbenligiray) [inline api3-whitepaper.tex]: I think "will" is more accurate here, as an outage that doesn't result in a payout won't penalize stakers

File: api3-whitepaper.tex [hunk 2]
```diff
@@ -784,44 +784,44 @@ \subsection{dAPI Insurance}
 
 It has been shown that insurance usage not only correlates with macroeconomic growth, but is also a cause of it~\cite{outreville:2013}.
 Nevertheless, insurance is sorely underutilized in the blockchain space.
-One of the main reasons of this is that insurance naturally requires a third party to resolve insurance disputes, and using a mutually-trusted third party for this purpose is against the ethos of decentralization.
-However, the emergence of Kleros~\cite{kleros:2019}, a general purpose on-chain dispute resolution protocol, allows trustless insurance products to be built.
+One of the main reasons of this is that insurance naturally requires a centralized third party to resolve insurance disputes, and using a mutually-trusted third party for this purpose is against the ethos of decentralization.
```
Comment (bbenligiray) [inline api3-whitepaper.tex]: The third party isn't required to be centalized and Kleros is an example of a decentralized third party, so I don't get this addition

File: api3-whitepaper.tex [hunk 3]
```diff
@@ -929,6 +929,8 @@ \section{Conclusion}
 \section{Whitepaper Versions}
 \label{sec:versions}
 
+\href{https://github.com/api3dao/api3-whitepaper/releases/tag/v1.0.3}{v1.0.3} -- Burak Benligiray, Erich Dylus, June 2022.
```
Comment (bbenligiray) [inline api3-whitepaper.tex]: You can swap the order of the authors, you did all the work

**Review by bbenligiray — COMMENTED** [review]
A few comments before we merge, looks great otherwise. We should probably do another release after coverage is out to be more specific and accurate about the implementation.
After it's merged I can rebuild the PDF and fix any potential wonkyness in typesetting.

File: api3-whitepaper.tex (hunk 1 above)
Comment (ErichDylus) [reply api3-whitepaper.tex]: agreed, reverted this to "will"

File: api3-whitepaper.tex (hunk 2 above)
Comment (ErichDylus) [reply api3-whitepaper.tex]: My thought was that true insurance (i.e. provided by a properly licensed and collateralized regulated entity) is necessarily centralized, but this addition is unnecessary so I'll take it out

Comment (bbenligiray) [reply api3-whitepaper.tex]: Ah okay, makes sense. I think this won't be clear to the reader unless we give the context of the technicality on which we're not calling this insurance so better to leave it out.

**Pull request merged**